src/console/controllers/GcController.php

```diff
@@ -31,7 +31,7 @@ class GcController extends Controller
     /**
      * @var bool Whether all soft-deleted items should be deleted, rather than just
      * the ones that were deleted long enough ago to be ready for hard-deletion
-     * per the `softDeleteDuration` config setting.
+     * per the <config3:softDeleteDuration> config setting.
```

Comment:
We should avoid special link syntaxes in these docblocks because they won’t get parsed in `help` command output.

Reply:
Ah, that’s right. Will revise. 👍